udata_recommendations/views.py

```diff
@@ -8,14 +8,30 @@
 blueprint = Blueprint('recommendations', __name__, template_folder='templates')
 
 
+NB_RECOMMENDATIONS = 2
```

Comment:
Should this be configurable? I kept the old value (it was in the HTML template directly before).

Reply:
yes, `settings.py` FTW :-) I think we can go up to 4 by default now 💪